simulator/include/simulator.h

```diff
@@ -8,6 +8,7 @@
 
 #include <SFML/Graphics.hpp>
 #include <SFML/Audio/SoundRecorder.hpp>
+#define MICRO_PHONE_H
```

Comment:
The simulation of the microphone is pretty basic for now:
```c++
namespace microphone {

static constexpr float silenceLevelDb = -57;
static constexpr float highLevelDb = 80;

static float sound_level;

float get_sound_level_Db() {
  return sound_level;
}

// ... SFML code processing desktop microphone audio ...
```

These conflict with the ones defined in `microphone.h` however, as per good practices, this file is guarded:
```
% grep -rn MICRO_PHONE_H src
src/system/physical/MicroPhone.h:1:#ifndef MICRO_PHONE_H
```

Hence defining `MICRO_PHONE_H` explicitly in `simulator.h` is a way to avoid conflict with `microphone.h` potentially included elsewhere, just as we now try to simulate the whole lamp at once. It is not a definitive solution, and is subject to change, as the way the simulation is done is subject to change as the software evolves.

Marking this as solved, as it seems to me to be a temporary quick fix of a minor problem.